### PR TITLE
Fixed TTyS3

### LINKED ISSUE #79: TTyS3 not working
Hi guys. Yesterday I pluged my GPS to TTyS3 and it did not connect at all. It works on TTyS1 and TTyS2.  TTyS3 works with Armstrong image.
Do you know if someone else is having the same issue?
Do these ports get enabled with cubieborard2-cubiescreen.fex file?

### PR DESCRIPTION
**Pull request by iglesiasmarianod**
Changed TTyS3 pinout to match schmatics. Fixes #79

### CODE REVIEW

File: recipes-kernel/linux/linux-mainline/openvario-common.dts [hunk 1]
```diff
@@ -253,6 +253,11 @@ cubiets@5c {
 		function = "gpio_in";
 		bias-pull-up;
 	};
+
+	uart5_ph_pins: uart5-ph-pins {
+		pins = "PI10", "PH7";
+		function = "uart5";
+	};
```
[the remark this answers is not in the document]
Comment (iglesiasmarianod) [reply recipes-kernel/linux/linux-mainline/openvario-common.dts:260]: Hi @kedder .
Tried this:
```
	&uart5-ph-pins {
		pins = "PI10", "PH7";
		function = "uart5";
	};
};

&pwm {
	pinctrl-names = "default";
	pinctrl-0 = <&pwm0_pin>;
	status = "okay";
};
```
and this

```
&uart5-ph-pins {
	pins = "PI10", "PH7";
	function = "uart5";
};

&pwm {
```
first one inside &pio, the second one on top level.
Both exit with syntax error. I can't figure out why.

[the remark this answers is not in the document]
Comment (kedder) [reply recipes-kernel/linux/linux-mainline/openvario-common.dts:260]: Rigt, please try exactly as I suggested: on top level, using name with underscores (not dashes).  If still doesn't work, leave it as is.

```
&uart5_ph_pins {
    pins = "PI10", "PH7";
}
```

[the remark this answers is not in the document]
Comment (iglesiasmarianod) [reply recipes-kernel/linux/linux-mainline/openvario-common.dts:260]: jajaja did not see the underscores!! sorry!
Let me try it!

[the remark this answers is not in the document]
Comment (kedder) [reply recipes-kernel/linux/linux-mainline/openvario-common.dts:260]: The name with underscore (`uart5_ph_pins`) is the id (aka "label") of the device tree node. It is the one that is referenced in `uart5` node. The thing with dashes is something else. I'm not completely sure what it is used for, but it does not work as a reference to a node.

[the remark this answers is not in the document]
Comment (iglesiasmarianod) [reply recipes-kernel/linux/linux-mainline/openvario-common.dts:260]: Did not realized they were underscores till you told me.
Did not see them even in the other definitions!

[the remark this answers is not in the document]
Comment (iglesiasmarianod) [reply recipes-kernel/linux/linux-mainline/openvario-common.dts]: It worked! Thanks! I pushed the change right now.

**Comment by linuxianer99** [comment]
@iglesiasmarianod Have you checked if it is common for all hardware platforms ?
Stefly 5,7", original 7" and 4,3RGB ?

**Comment by iglesiasmarianod** [comment]
Hi @linuxianer99. Checked the other .fex files and 4.3RGB is different. 070-LVDS, 070PQ and 5.7-LVDS share same .fex pinout.
AFAIK Stefly's devices use the same LVDS images. It would be great if someone can test it. Don't have the schematics for his boards.
Should I move the changes to the specific .dts files instead of common?

**Comment by linuxianer99** [comment]
@iglesiasmarianod : Think it makes sense in this case to move to the specific DTS.
Even we have them already in place for the different panel, etc ...

I have 7" variant here on my desk for testing ..

**Comment by iglesiasmarianod** [comment]
Great. Let me change it and push it again.

**Pull request merged**